src/tiled/templategroupdocument.cpp

```diff
@@ -45,7 +45,7 @@ TemplateGroupDocument::TemplateGroupDocument(TemplateGroup *templateGroup, const
     : Document(TemplateGroupDocumentType, fileName)
```

Comment:
I think so. The template group should always contain the file name, so the constructor arguments can be the template group only.